backend/tournesol/tests/test_api_unconnected_entities.py

```diff
@@ -50,15 +52,18 @@ def setUp(self):
             entity_2=video_5,
         )
 
-    def test_not_authenticated_cannot_show_unconnected_entities(self):
+    def test_anon_get_401(self):
+        """
+        An anonymous user must not be able to list its unconnected entities.
+        """
         response = self.client.get(
             f"{self.user_base_url}/{self.video_source.uid}/",
             format="json",
         )
 
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
 
-    def test_without_uid_show_not_found(self):
+    def test_auth_get_404_with_no_uid(self):
```

Comment:
Minor: should it return Bad Request instead of Not Found in this case?

Reply:
I was surprised too, I need to check how the rest of the API behaves in this kind of case.